### DISCUSSION TITLE
Should we meet in September for an SDC-tools User Group Workshop?

### DISCUSSION THREAD

**Discussion by violetacln** [sdcTools/UserSupport#253] [User Group]
This is to remind you to vote  whether to organize a new SDC-tools Users' Workshop in Wiesbaden, in September 2023, accompanying the UNECE conference on SDC (26-28 September, https://indico.un.org/event/1002843/ ).

Please, note that this Workshop would be an addition to our regular User Group meetings which always take place at the beginning of the year. Please, feel free to comment and propose interesting topics or formats for the workshop, too.

Please, vote, by going to the link below (NOTE: you need to be signed into your github account beforehand):
https://github.com/orgs/sdcTools/discussions/249

Looking forward to your decision!

**Comment by AncoHundepool** [comment]
Dear Violeta,
   
   Great that you are organising this user group.
   If it takes place and I will be at the UNECE, l will attend, but I am not influencing the agendas of the active workers.
   So I will not vote.
   
   Best regards,
   
  
  
   Anco
   
  
  
   
    Op 21-06-2023 15:42 schreef Violeta Calian ***@***.***>:
   
   
    
   
   
    
   
   
   This is to remind you to vote whether to organize a new SDC-tools Users' Workshop in Wiesbaden, in September 2023, accompanying the UNECE conference on SDC (26-28 September, https://indico.un.org/event/1002843/ ).
   Please, note that this Workshop would be an addition to our regular User Group meetings which always take place at the beginning of the year. Please, feel free to comment and propose interesting topics or formats for the workshop, too.
   Please, vote, by going to the link below (NOTE: you need to be signed into your github account beforehand):https://github.com/orgs/sdcTools/discussions/249
   Looking forward to your decision!
